reason-parser/src/reason_parser.mly

```diff
@@ -1200,7 +1201,7 @@ _use_file:
 parse_core_type:
     core_type EOF
       {
-        let core_loc = mklocation $startpos($1) $endpos($1) in
+        let core_loc = mklocation $symbolstartpos $endpos($1) in
```

Comment:
Ah yes but I think in this case it is not going to do anything because core_type is not inlined and not nullable, so `$symbolstartpos == $startpos(1)` (not nullable means it cannot be absent, not inlined means it is not going to be expanded in one or more producers that could be nullable themselves).
It is correct nonetheless, so that's fine :).